lib/ex_plasma.ex

```diff
@@ -78,19 +78,24 @@ defmodule ExPlasma do
     ...>      }
     ...>    ],
     ...>    sigs: [],
-    ...>    tx_data: <<0>>,
+    ...>    tx_data: 0,
```

Comment:
I see that typespec for tx_data is any() `https://github.com/omgnetwork/ex_plasma/blob/55a72267adfd9fd5d008b010fca2816db018cc56/lib/ex_plasma/transaction.ex#L33

so it can be both integer and binary?

Reply:
The thing is that it's not used for now so it's `0` and we don't really know how it'll be used in the future so I think we just put any(). But I can update to non_neg_integer() for now to reflect the current state.